charts/gsp-cluster/templates/02-gsp-system/service-operator/crd/storage.govsvc.uk_imagerepositories.yaml

```diff
@@ -0,0 +1,99 @@
+{{- if .Values.global.runningOnAws }}
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  name: imagerepositories.storage.govsvc.uk
+spec:
+  group: storage.govsvc.uk
+  names:
+    kind: ImageRepository
+    plural: imagerepositories
+  scope: ""
+  validation:
+    openAPIV3Schema:
+      description: ImageRepository is the Schema for the ImageRepository API
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: ImageRepositorySpec defines the desired state of ImageRepository
+          properties:
+            aws:
```

Comment:
aws specific config (in this is case it will likely be things like tag mutability and lifecycle policy) ... just following the pattern from the other resources for now